provision/acc_provision/templates/provision-config.yaml

```diff
@@ -146,6 +146,7 @@ registry:
   # enable_opflex_agent_reconnect: True                 # default is False, set it as True to enable opflex agent reconnect after vm migration
   # opflex_agent_statistics: False                      # default is True, set this to False if statistics load is very high
   # add_external_contract_to_default_epg: True          # override if needed, default is False
+  # apic_conn_retry_limit: 5                            # override if needed, default is 5
```

Comment:
can you pls give a description on what the parameter is used for

Reply:
Updated the description Akhila, thank you.